src/track/cueinfoimporter.cpp

```diff
@@ -0,0 +1,79 @@
+#include "track/cueinfoimporter.h"
+
+namespace mixxx {
+
+CueInfoImporter::CueInfoImporter() {
+}
+
+CueInfoImporter::CueInfoImporter(const QList<CueInfo>& cueInfos)
+        : m_cueInfos(cueInfos) {
+}
+
+double CueInfoImporter::getTimingOffsetMillis(
+        const QString& filePath,
+        const audio::SignalInfo& signalInfo) const {
+    Q_UNUSED(filePath);
+    Q_UNUSED(signalInfo);
+    return 0;
+};
+
+void CueInfoImporter::append(const CueInfo& cueInfo) {
+    m_cueInfos.append(cueInfo);
+}
+
+void CueInfoImporter::append(const QList<CueInfo>& cueInfos) {
+    m_cueInfos.append(cueInfos);
+}
+
+int CueInfoImporter::size() const {
+    return m_cueInfos.size();
+}
+
+bool CueInfoImporter::isEmpty() const {
+    return m_cueInfos.isEmpty();
+}
+
+QList<CueInfo> CueInfoImporter::getCueInfosWithCorrectTiming(
+        const QString& filePath,
+        const audio::SignalInfo& signalInfo) {
+    QList<CueInfo> cueInfos;
+
+    // Do not calculate offset if we don't have any cues to import
+    if (m_cueInfos.isEmpty()) {
+        cueInfos.append(m_cueInfos);
+        m_cueInfos.clear();
```

Comment:
`m_cueInfos` is already empty, so is this line needed?

Reply:
woops, removed.